api/src/main/java/org/pmiops/workbench/initialcredits/InitialCreditsExpirationServiceImpl.java

```diff
@@ -130,49 +143,46 @@ private void checkExpiration(DbUser user) {
     DbUserInitialCreditsExpiration userInitialCreditsExpiration =
         user.getUserInitialCreditsExpiration();
     if (null != userInitialCreditsExpiration
```

Comment:
you can also probably have an early return, something like 
`if (userInitialCreditsExpiration == null || userInitialCreditsExpiration.getExpirationCleanupTime() != null) {
        return;
    }`

Reply:
Good call. I added this.